tensorflow_datasets/image/celebahq.py

```diff
@@ -116,23 +120,25 @@ def _info(self):
 
   def _split_generators(self, dl_manager):
     """Returns SplitGenerators."""
-    image_tar_file = os.path.join(dl_manager.manual_dir,
```

Comment:
Why modifying the logic here ?

Reply:
I did not modify the base logic.
Just updated it to support all types of archives `zip`, `tar`, etc.

Reply:
I understand the motivation but this add unnecessary complexity to the code. Please revert the change.

Reply:
The drive files contain only `.zip` files.
So, should I change to `tar` to `zip`?

Reply:
Yes

Reply:
From what I understand, thos zip files re not the official source, so we should not use them.